### DISCUSSION TITLE
How can I add a Combobox element when customizing a Theme

### DISCUSSION THREAD

**Discussion by l0rd** [wixtoolset/issues#8113] [Questions] [answered]
Hi 👋 ,

I am trying to add a ['Combobox'](https://wixtoolset.org/docs/schema/thmutil/combobox/) to the 'Install' Page of a BootstrapperApplication (`WixStandardBootstrapperApplication.HyperlinkSidebarLicense`) using a Theme:

```
<Theme xmlns="http://wixtoolset.org/schemas/thmutil/2010">
  (...)
  <Page Name="Install">
    (...)
    <Combobox Name="MachineProvider" X="185" Y="250" Width="-11" Height="21" TabStop="yes" FontId="6" Sorted="yes" DropDownList="yes" HideWhenDisabled="yes" Property="MACHINE_PROVIDER">
      <ListItem Text="Hyper-V" Value="hyperv" />
      <ListItem Text="WSL" Value="wsl" />
    <Combobox/>
    (...)
  </Page>
</Theme>
```

When I do that, the generated `setup.exe` exits right away without logging anything. 

I am using Wix Toolset v3.14.

Is there a way to get more logs when I am running `setup.exe`?
Is there a command line tool to validate the Theme?
How is ThmViewer.exe supposed to be used (CLI parameters, pre-requisites)? (It exits right away if I try to run it on Windows 11)

Thank you

**Comment by l0rd** [comment, marked as answer]
Although I didn't get any answer I am closing this one as:
- [This doesn't look to be possible using WiX v3](http://lists.wixtoolset.org/pipermail/wix-users-wixtoolset.org/2017-August/005295.html)
- [WiX v4 Thmutil schema includes radio buttons](https://wixtoolset.org/docs/schema/thmutil/radiobutton/) and that's covered by [unit tests](https://github.com/wixtoolset/wix/blob/ca8a55ed44bd739e5eb7d77fbcdb3eba3ba0fee8/src/test/burn/TestData/Manual/BafThmutilTesting/theme/BafThmUtilTestingTheme.xml#L175) (radio buttons were what I was looking for in the first place)

**Reply by robmen** [reply]
We're leaving questions open since they seem to be more discoverable that way. I also marked the answer to help people looking for similar information later.
